code-experiments/build/matlab/my_optimizer.m

```diff
@@ -6,5 +6,20 @@ function my_optimizer (problem, lower_bounds, upper_bounds, budget)
     cocoEvaluateFunction(problem, x);
     if cocoProblemGetNumberOfConstraints(problem) > 0
         cocoEvaluateConstraint(problem, x)
+        cocoProblemGetName(problem)
     end
 end
+
+%
+% asma: The code below calls MATLAB solver, fmincon, on bbob-constrained testbed
+% cons is defined in cons.m
+%
+
+% function my_optimizer (problem, lower_bounds, upper_bounds, budget)
```

Comment:
Shouldn't this go into a separate file without the comments? That allows the user to easily exchange the two solvers without uncommenting code (and without having working code within a comment).